### ISSUE TITLE
eupd step failed with NetCDF: HDF error?

### ISSUE THREAD

**Issue by XuLi-NOAA** [NOAA-EMC/GSI#357]
In one cycle run, oprcst2, which is starting from 2021110400 with ICs from oprcst10, eupd failed at 2021110418 cycle.
The difference between oprcst2 and oprcst10 is the in situ observation QC, it is looser for opecst2. 

oprcst10 is running well, just opecst2 has the above issue. 

After recompile of GSI, 2021110418 eupd went through, but failed at 2021110500 eupd. 

This happened around April 1, 2022. Then I tried again, 2021110500 eupd went through and oprcst2 is running well (so far). 

The failure message:

 **   ERROR: NetCDF: HDF error
 ** Failed to read NetCDF4.

It has been found out the eupd failed while reading satellite data, specifically, reading viirs npp and j1, which are added the first time.
 
**In summary, there was a NetCDF related error while reading satellite diag files in EnKF, but then the error gone without any change.** 

On hera, in one of my utility, the similar error, NetCDF: HDF error, happened on April 4, 2022. And then, it works again on April 5, after a system maintenance. I asked Hera helpdesk if they have any ideas why, Raghu said:"We had a filesystem issue on scratch1 and that was taken care of during yesterday's maintenance."

**Comment by RussTreadon-NOAA** [comment]
@XuLi-NOAA , have you recompiled global_enkf.x in debug node and rerun a case with the NetCDF error?

**Comment by XuLi-NOAA** [comment]
> @XuLi-NOAA , have you recompiled global_enkf.x in debug node and rerun a case with the NetCDF error?

@RussTreadon-NOAA : Not yet. Yes, it is necessary to find out what happened exactly.

**Comment by aerorahul** [comment]
There have been several reports of this issue when parallel netcdf is used to write the model backgrounds.
I believe it is fixed in the current `develop` of the global-workflow by switching to use serial netcdf.
It is also being investigated as part of series of issues exposed in the WCOSS2 transition.

Regardless, I do not think this is an issue in the GSI or EnKF, but rather an issue with the HDF5 library.  A suggestion (from GDIT) was made to upgrade to a later HDF5 library or use serial netcdf.  The latter is being used for now.

**Comment by RussTreadon-NOAA** [comment]
@XuLi-NOAA , can this issue be closed?

**Comment by XuLi-NOAA** [comment]
Yes, it can be closed.
